test/testdrive/temporary.td

```diff
@@ -104,6 +117,8 @@ unknown catalog item 'double_temp_v'
 
 > SELECT * FROM mz_indexes WHERE name = 'foo_primary_idx'
 
+> SELECT * FROM mz_indexes WHERE name = 'temp_t_primary_idx'
+
```

Comment:
```SHOW CREATE TABLE LIKE``` does not pick temporary tables.

```
pstoev=> create temporary table t1 (f1 integer);
CREATE TABLE

pstoev=> show tables like 't1';
 name 
------
(0 rows)

```

Reply:
I believe that all of our `SHOW` commands only show items in the current (or given schema) by default. For some `SHOW` commands, we have an `EXTENDED` option that will include items outside of the current schema, as well.

Do you think that implementing `SHOW EXTENDED TABLES` like this would be sufficient? Or should `SHOW TABLES` always include a connection's temporary tables?
```
pstoev=> create temporary table t1 (f1 integer);
CREATE TABLE

pstoev=> show extended tables like 't1';
 name 
------
(0 rows)
```

Reply:
EXTENDED is good enough, thank you.

Reply:
Perfect! Created an issue for that here: https://github.com/MaterializeInc/materialize/issues/5593